app/services/transient_registration_cleanup_service.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class TransientRegistrationCleanupService < ::WasteExemptionsEngine::BaseService
+  def run
+    transient_registrations_to_remove.each(&:destroy)
```

Comment:
I've not tested directly what the performance implications are, but I have tested locally and this works in the same way

```suggestion
    transient_registrations_to_remove.destroy_all
```

I'm assuming that this would be more performant that iterating through each result and calling `destroy()`. Plus I think the intent is slightly more clearer. What do you think?